packages/app-project/src/screens/ClassifyPage/components/ClassifierWrapper/ClassifierWrapperConnector.spec.js

```diff
@@ -41,10 +41,6 @@ describe('Component > ClassifierWrapperConnector', function () {
         expect(wrapper.props().recents).to.equal(mockStore.user.recents)
       })
 
-      it('should include your personal stats', function () {
-        expect(wrapper.props().yourStats).to.equal(mockStore.user.personalization)
-      })
```

Comment:
`user.personalization` does not always equal "your personal stats". Stats used to be a part of the UserPersonalization store, but that's not the case in this PR. Specs that equate personalization to stats have been removed.